bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml

```diff
@@ -0,0 +1,302 @@
+              <lb ed="#B"/>suis In quamcumque Ciuitatem intra<lb ed="#B"/>ueritis 
+              et susceperint vos man<lb ed="#B"/>ducate 
+              que apponuntur vobis 
+              <lb ed="#B"/>et cetera In quamcumque ciuitatem in <unclear>curenvuf</unclear>  
+              <lb ed="#B"/>et non receperint vos exeuntes in 
+              <lb ed="#B"/>plateis eis dicite etiam puluerem 
+              <lb ed="#B"/>qui adhesit vobis de ciuitate viam
+              <lb ed="#B"/>extergimus in vos tamen hoc scitote 
+              <lb ed="#B"/>quia appropinquabit regnum dei</quote>
+            </p>
+            <p xml:id="bHY6yh-de5520-d1e404">
+              <lb ed="#B"/>¶ <ref xml:id="bHY6yh-de5520-Rd1e445">capitulo 16</ref> <quote xml:id="bHY6yh-de5520-Qd1e408">homo quidam erat diues 
+              <lb ed="#B"/>et induebatur purpura et bysso 
+              <lb ed="#B"/>et epulabatur cottidie splendi
+              <lb ed="#B"/>de / et erat quidam inendit nomine
+              <lb ed="#B"/>lazarusut qui iacebat ad iannam 
+              <lb ed="#B"/>eius vlceribus plenus cupieus
+              <lb ed="#B"/>demensa diuitis et nemo illi dabat
+              <lb ed="#B"/>sed eccanes veniebant et ligue<lb ed="#B"/>baut
+              vlceri eius factum est ac 
+              <lb ed="#B"/>ut <unclear>moretetimin</unclear> paupere seu men<lb ed="#B"/>dicus
+              et <unclear>pertaretur</unclear> ab angelis
+              <lb ed="#B"/>in synum abrahe mortuus est
+              <lb ed="#B"/>autem et diues et seperultus est in
+              <lb ed="#B"/>inferno eleuans autem oculos suos
+              <lb ed="#B"/>cum esset in tormentis vidit abraham
+              <lb ed="#B"/>alonge et lazarum in synu cuis et 
+              <lb ed="#B"/>ipse clamans dixit pare abra<lb ed="#B"/>ham
+              miserere mei et initte la<lb ed="#B"/>zarum
+              digiti sui in aquam et re<lb ed="#B"/>frigeter 
+              ligivam meam quia cruci<lb ed="#B"/>ar
+              in hac flamma et dixit illi
+              <lb ed="#B"/>abraham fili recordare quia re<lb ed="#B"/>cepisti 
+              bona in vita tua et laza<lb ed="#B"/>rus
+              <unclear>suulr</unclear> in ala hunc autem hic
+              <lb ed="#B"/>ussoldatur <unclear>tu nero</unclear> crucilitis et in
+              <lb ed="#B"/>hiis <unclear>onunly</unclear> vitus nos et uos chaos
+              <lb ed="#B"/>magnum firmatum est vt <unclear>lrius</unclear>
+              <lb ed="#B"/>volunt huic cursite aduos <subst><add place="margin-right">non</add></subst> possunt
+              <lb ed="#B"/>nequam inde nec huc transmeare
+              <lb ed="#B"/>et Aut Rogo ergo <unclear>prince</unclear> ut mittas
+              <lb ed="#B"/>eum in dominum patris mei / habeo enim
+            <pb ed="#B" n="10-v"/>
+            <cb ed="#B" n="a"/>
+              <lb ed="#B"/>quinque fratres vt testetur illis ne
```

Comment:
ideally the `cb` and `pb` element should be indented flush with the `lb` elements.
```suggestion
              <lb ed="#B"/>eum in dominum patris mei / habeo enim
              <pb ed="#B" n="10-v"/>
              <cb ed="#B" n="a"/>
              <lb ed="#B"/>quinque fratres vt testetur illis ne
```